docker/docker-compose.dev.yml

```
@@ -94,29 +93,29 @@ services:
-            
-            /opt/mssql-tools18/bin/sqlcmd -C -U sa -P $$MSSQL_SA_PASSWORD -Q "CREATE DATABASE llana;"
-            /opt/mssql-tools18/bin/sqlcmd -C -U sa -P $$MSSQL_SA_PASSWORD -l 30 -e -i /docker-entrypoint-initdb.d/mssql.sql
+                /opt/mssql-tools18/bin/sqlcmd -C -U sa -P $$MSSQL_SA_PASSWORD -Q "CREATE DATABASE llana;"
+                /opt/mssql-tools18/bin/sqlcmd -C -U sa -P $$MSSQL_SA_PASSWORD -l 30 -e -i /docker-entrypoint-initdb.d/mssql.sql
 
-            echo "Script Execution is complete. Waiting for MS SQL Process to terminate 🎉"
+                echo "Script Execution is complete. Waiting for MS SQL Process to terminate 🎉"
 
-            wait $$pid
\ No newline at end of file
+                wait $$pid
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)